notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<p>Maybe add an note about what <code>tf.keras.applications</code> and how it is similar/different to tf-hub.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Makes sense. I added an explanation on <code>tf.keras.application</code> vs tf-hub here.</p><p>Also, I think it is a good idea to incorporate <code>tf.keras.application</code> in Transfer Learning module. </p><p>Thanks!</p>

Reply:
<p>Nice explanation! </p>

Reply:
<p>Thank you!</p>